src/main/java/uk/gov/hmcts/dts/fact/exception/GlobalControllerExceptionHandler.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.dts.fact.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class GlobalControllerExceptionHandler {
+    @ExceptionHandler(NotFoundException.class)
+    ResponseEntity<String> slugNotFoundHandler(NotFoundException ex) {
```

Comment:
A small one here, the name `slugNotFoundHandler` assumes that all 404's being picked up with be related to slugs not being found, will that always be the case? It could be that a 404 is raised from elsewhere in the future, so simply calling it something akin to `notFoundExceptionHandler` might be good if thats true

Reply:
Yeah good point. notFounExceptionHandler would be better. I just copied this out from the existing code. I will change it